docs/config.md

```diff
@@ -2542,57 +2542,68 @@ Contains options for content, peer, and IPNS routing mechanisms.
+>   [`Routing.Routers`](#routingrouters). See [delegated-routing.md](delegated-routing.md).
 
-To force a specific Amino DHT-only mode, client or server, set `Routing.Type` to
-`dhtclient` or `dhtserver` respectively. Please do not set this to `dhtserver`
-unless you're sure your node is reachable from the public network.
+Default: `auto`
 
-When `Routing.Type` is set to `auto` or `autoclient` your node will accelerate some types of routing
-by leveraging [`Routing.DelegatedRouters`](#routingdelegatedrouters) HTTP endpoints compatible with [Delegated Routing V1 HTTP API](https://specs.ipfs.tech/routing/http-routing-v1/)
-introduced in [IPIP-337](https://github.com/ipfs/specs/pull/337)
-in addition to the Amino DHT.
+Type: `optionalString` (`null`/missing means the default)
+
+### `Routing.DelegatedRouters`
 
-When `Routing.Type` is set to `delegated`, your node will use **only** HTTP delegated routers and IPNS publishers,
-without initializing the Amino DHT at all. This mode is useful for environments where peer-to-peer DHT connectivity
-is not available or desired, while still enabling content routing and IPNS publishing via HTTP APIs.
-This mode requires configuring [`Routing.DelegatedRouters`](#routingdelegatedrouters) for content routing and
-[`Ipns.DelegatedPublishers`](#ipnsdelegatedpublishers) for IPNS publishing.
+An array of URL hostnames for delegated routers to be queried in addition to the Amino DHT when `Routing.Type` is set to `auto` (default) or `autoclient`.
+These endpoints must support the [Delegated Routing V1 HTTP API](https://specs.ipfs.tech/routing/http-routing-v1/).
 
-**Note:** `delegated` mode operates as read-only for content providing - your node cannot announce content to the network
-since there is no DHT connectivity. Content providing is automatically disabled when using this routing type.
+The special value `"auto"` uses delegated routers from [AutoConf](#autoconf) when enabled.
 
-[Advanced routing rules](https://github.com/ipfs/kubo/blob/master/docs/delegated-routing.md) can be configured in `Routing.Routers` after setting `Routing.Type` to `custom`.
+> [!TIP]
+> Delegated routing allows IPFS implementations to offload tasks like content routing, peer routing, and naming to a separate process or server while also benefiting from HTTP caching.
+>
+> One can run their own delegated router either by implementing the [Delegated Routing V1 HTTP API](https://specs.ipfs.tech/routing/http-routing-v1/) themselves, or by using [Someguy](https://github.com/ipfs/someguy), a turn-key implementation that proxies requests to other routing systems. A public utility instance of Someguy is hosted at [`https://delegated-ipfs.dev`](https://docs.ipfs.tech/concepts/public-utilities/#delegated-routing).
 
-Default: `auto` (DHT + [`Routing.DelegatedRouters`](#routingdelegatedrouters))
+Default: `["auto"]`
 
-Type: `optionalString` (`null`/missing means the default)
+Type: `array[string]` (URLs or `"auto"`)
```

Comment:
Can `auto` be combined with custom ones? For example, if I want to combine Mainnet + custom content routing?